dev-java/lz4-java/lz4-java-1.8.0.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source test"
+MAVEN_ID="org.lz4:lz4-java:1.8.0"
+
+inherit java-pkg-2 java-ant-2
+
+DESCRIPTION="LZ4 compression for Java"
+HOMEPAGE="https://github.com/lz4/lz4-java"
+SRC_URI="https://github.com/lz4/lz4-java/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# Tests require com.carrotsearch.randomizedtesting:junit4-ant,
+# which is yet to be packaged
+RESTRICT="test"
+
+CDEPEND="
+	app-arch/lz4
+"
+
+# The version requirement on mvel is strict; mvel-2.4.* generates source files
+# incorrectly, which causes build failure of this package
+DEPEND="
+	>=virtual/jdk-1.8:*
+	dev-java/cpptasks:0
+	~dev-java/mvel-2.3.2:0
```

Comment:
because of this i changed `dev-java/mvel-2.3.2` to slot `2.3`. otherwise this package would block usage of newer versions from other packages, once they use `mvel` as well.